package.json

```
@@ -24,5 +24,8 @@
             "components/*/typescript-*",
             "components/supervisor/frontend"
         ]
+    },
+    "resolutions": {
+        "sha.js": "2.4.12"
```

Not blocking (suggestion and question):

We could allow any patch version like so:
```suggestion
        "sha.js": "~2.4.12"
```

What version did we have previously? I ask because I'm not sure how large of a jump this is.

---

Ah, good point, thank you for sharing!